src/gtk/app.cpp

```diff
@@ -31,6 +31,9 @@
 #include <gtk/gtk.h>
 #include "wx/gtk/private.h"
 
+#if wxUSE_MIMETYPE
```

Comment:
There is also check for this inside `wx/gtk/mimetype.h`, one of them seems unnecessary.